types/jsonwebtoken/index.d.ts

```diff
@@ -74,7 +74,7 @@ export interface DecodeOptions {
 }
 
 export interface VerifyCallback {
-    (err: JsonWebTokenError | TokenExpiredError | NotBeforeError, decoded: any): void;
+    (err: JsonWebTokenError | NotBeforeError | TokenExpiredError, decoded: Object | string): void;
```

Comment:
If we've enabled a version on DT that means it's OK to update to that version.
But beware that `object | string | JsonWebTokenError | NotBeforeError | TokenExpiredError` is just `object | string` because everything else falls under `object`.